build.gradle.kts

```diff
@@ -71,23 +72,24 @@ tasks {
         untilBuild.set(properties("pluginUntilBuild"))
 
         // Extract the <!-- Plugin description --> section from README.md and provide for the plugin's manifest
-        pluginDescription.set(
-            file("README.md").readText().lines().run {
-                val start = "<!-- Plugin description -->"
-                val end = "<!-- Plugin description end -->"
-
-                if (!containsAll(listOf(start, end))) {
-                    throw GradleException("Plugin description section not found in README.md:\n$start ... $end")
-                }
-                subList(indexOf(start) + 1, indexOf(end))
-            }.joinToString("\n").let { markdownToHTML(it) }
+        pluginDescription.set(provider {
```

Comment:
It would be better to use `providers.fileContents()` here for the configuration cache to better understand the inputs

Reply:
Refactored in 467c9db1bf8d28522bec687ec04a60dc771fec47